bls_thresholdsign.go

```diff
@@ -339,20 +347,20 @@ func (s *blsThresholdSignatureInspector) TrustedAdd(orig int, share Signature) (
 }
 
 // VerifyAndAdd verifies a signature share (same as `VerifyShare`),
-// and may or may not add the share to the local pool of shares.
-// This function is thread safe and locks the internal state.
+// and attempts to add the share to the local pool of shares.
+// This function is thread-safe and locks the internal state.
 //
-// The share is only added if the signature is valid, the signer index is valid and has not been
-// added yet. Moreover, the share is added only if not enough shares were collected.
-// Boolean returns:
-//   - First boolean output is true if the share is valid and no error is returned, and false otherwise.
-//   - Second boolean output is true if enough shares were collected and no error is returned, and false otherwise.
+// The share is only added if the signature is valid, the signer index is valid,
+// and has not been added yet. Moreover, the share is added only if not enough shares were collected.
 //
-// Error returns:
-//   - invalidInputsError if input index is invalid. A signature that doesn't verify against the signer's
+// Returns:
+//   - First boolean: true if the share is valid and no error is returned, false otherwise.
+//   - Second boolean: true if enough shares were collected and no error is returned, false otherwise.
+//   - Error:
+//   - invalidInputsError: if input index is invalid. A signature that doesn't verify against the signer's
 //     public key is not considered an invalid input.
-//   - duplicatedSignerError if signer was already added.
-//   - other errors if an unexpected exception occurred.
+//   - duplicatedSignerError: if signer was already added.
+//   - other errors: if an unexpected exception occurred.
 func (s *blsThresholdSignatureInspector) VerifyAndAdd(orig int, share Signature) (bool, bool, error) {
```

Comment:
```suggestion
func (s *blsThresholdSignatureInspector) VerifyAndAdd(orig int, share Signature) (shareIsValid bool, enoughSharesCollected bool, err error) {
```
Could also name the return values in the signature